mmv1/templates/terraform/examples/discoveryengine_chat_engine_basic.tf.erb

```diff
@@ -0,0 +1,28 @@
+resource "google_discovery_engine_data_store" "data_store" {
+  location                    = "global"
+  data_store_id               = "<%= ctx[:vars]['data_store_id'] %>"
+  display_name                = "tf-test-structured-datastore"
+  industry_vertical           = "GENERIC"
+  content_config              = "NO_CONTENT"
+  solution_types              = ["SOLUTION_TYPE_CHAT"]
+}
+
+resource "google_discovery_engine_chat_engine" "<%= ctx[:primary_resource_id] %>" {
+  engine_id = "<%= ctx[:primary_resource_id] %>"
+  collection_id = "default_collection"
+  location = "global"
+  display_name = "test desplay name"
+  industry_vertical = "GENERIC"
```

Comment:
you could set this to the data store's industry_vertical since they have to match.

```suggestion
  industry_vertical = google_discovery_engine_data_store.data_store.industry_vertical
```